pkg/liqo-controller-manager/namespaceMap-controller/manage_remoteNamespaces.go

```diff
@@ -58,49 +59,58 @@ func (r *NamespaceMapReconciler) deleteRemoteNamespace(clusterID, remoteName str
 }
 
 // This function checks if there are Namespaces that have to be created or deleted, in accordance with DesiredMapping
-// and CurrentMapping fields.
-func (r *NamespaceMapReconciler) manageRemoteNamespaces(nm *mapsv1alpha1.NamespaceMap) error {
+// and CurrentMapping fields. It updates also NamespaceOffloading status in according to the remote Namespace Status.
+func (r *NamespaceMapReconciler) manageRemoteNamespaces(nm *mapsv1alpha1.NamespaceMap, requeue *bool) error {
```

Comment:
In my opinion, it's better to use a second return argument instead of a reference parameter, since it's more clear and readable. What do you think about that?

Reply:
yes it's a great advice, thank you.